src/main/resources/graphql/service/mutation.graphqls

```diff
@@ -89,13 +125,6 @@ input CreateQuizInput {
     If this is null or not set, the behavior is the same as if it was equal to the number of questions.
     """
     numberOfRandomlySelectedQuestions: Int @Positive
-
-    """
-    List of multiple choice questions.
-    """
-    multipleChoiceQuestions: [CreateMultipleChoiceQuestionInput!]! = []
```

Comment:
@Frontend would this be fine for you? Creating the questions together with the quiz is extremly complicated, for you and for us, because the data structure is different for each question type

Reply:
you just tagged some organization lol

Reply:
what the fuck

Reply:
@v-morlock should know more, because he took over the implementation of the lecturer site, where this would have been used. Personally, I think it makes sense now if we have the general information only in this input (CreateQuizInput)

Reply:
jo